ui/src/app/images/views/ImageList/ImageList.tsx

```diff
@@ -35,7 +38,13 @@ const ImagesList = (): JSX.Element => {
           security fixes.
         </Notification>
       )}
-      <UbuntuImages />
+      {ubuntu && (
+        <>
+          <UbuntuImages />
+          <hr />
+          <OtherImages />
```

Comment:
I think the `ubuntu` and `otherImages` data should be loaded at the same time, but might there be no `otherImages`? In which case we wouldn't want to show this section?